src/System.IO.UnmanagedMemoryStream/tests/ArrayHelpers.cs

```diff
@@ -66,6 +64,14 @@ public bool Equals(T[] x, T[] y)
                 }
                 for (int i = 0; i < x.Length; i++)
                 {
+                    if (x[i] == null)
+                    {
+                        if (y[i] == null)
+                        {
+                            return true;
```

Comment:
This return statement changes the semantics of the method. It should actually be `continue;`.